rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java

```diff
@@ -30,9 +30,8 @@
   public static final String URL_VERSION = PRE + "version";
   public static final String URL_TASKS = PRE + "tasks";
   public static final String URL_TASKS_ID = URL_TASKS + "/{taskId}";
-  public static final String URL_TASK_GET_POST_COMMENTS = URL_TASKS_ID + "/comments";
```

Comment:
Is the removal of this url breaking?

Reply:
I'm not removing anything. I'm just refactoring...